examples/visualization/visualization.ipynb

```diff
@@ -261,7 +267,11 @@
   {
    "cell_type": "code",
    "execution_count": 9,
-   "metadata": {},
+   "metadata": {
+    "tags": [
+     "nbval-skip"
+    ]
+   },
```

Comment:
Maybe in this PR, maybe in a different one, could you add some sort of early "example standards" to the developer docs that links to the full docs for invisible nbval tags, and recommends when to use them?